test/unittests/mongoBackend/mongoGetSubscriptions_test.cpp

```diff
@@ -96,12 +96,12 @@ static void prepareDatabaseV1Subs(void) {
                                                        "type" << "ONCHANGE" <<
                                                        "value" << BSON_ARRAY("AX2" << "AY2")
                                                        )) <<
-                        "throttling" << 5.0
+                        "throttling" << (long long) 5
                         );
 
     BSONObj sub3 = BSON("_id" << OID(SUB_OID3) <<
-                        "expiration" << 20000000 <<
-                        "lastNotification" << 25000000 <<
+                        "expiration" << (long long) 20000000 <<
```

Comment:
>  and rely in our "read long/int, write long" temporal fix.

Actually, I'm not sure about this... this is "expiration" (not "lastNotiication") and probably the current logic may work.